mock-plugin/target/apidocs/allclasses-index.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE HTML>
```

Comment:
why you have included the target files ?

Reply:
Removed